platform/darwin/src/MGLOfflinePack.mm

```diff
@@ -4,6 +4,8 @@
 #import "MGLOfflineRegion_Private.h"
 #import "MGLTilePyramidOfflineRegion.h"
 #import "MGLTilePyramidOfflineRegion_Private.h"
+#import "MGLTileListOfflineRegion.h"
```

Comment:
I was going to nit about alphabetizing the imports, but they're already screwed up :P